docs/getting-started.md

```diff
@@ -158,7 +158,7 @@ In the `Program.Main`, we configured the app with the basic feature set.
 ```
 
 To take advantage of many more additional features, such as
-[debug & parse directives](directives.md),
+[debug](debug-directive.md) & [parse](parse-directive) directives,
```

Comment:
@drewburlingame We have a problem currently. 
When we do a link like this: `[parse](parse-directive)` and forget to suffix `.md` it results into an invalid link and causes 404 and our build can not catch those invalid links. :(

Reply:
We might need another pass in a separate PR to fix such things. May be a regex search or something.

Reply:
yeah, I missed that.  I'm not sure how though since I was running a local build and watching other changes happen after that.  Mkdocs usually stops building when this type of error happens and I have to fix it and restart.